src/lib/index.ts

```diff
@@ -94,7 +99,7 @@ const main = async () => {
           let shouldCommentPr = false;
           let noBaseline = false;
           
-          if (snykDeltaResults.result > 0 && ghPRNumber) {
+          if (snykDeltaResults.result === 1 && ghPRNumber) {
```

Comment:
Why is this change required? What does it do or not do?

Reply:
Like that we send a comment on the PR only if there are issues. if snyk-delta result with code 2 is if snyk-delta failed and we are sending an comment with the generic title : 'Vulnerabilities report for commit number...'

Reply:
Ok